### DISCUSSION TITLE
How do I report a ports error?

### DISCUSSION THREAD

**Discussion by perrog** [microsoft/vcpkg#32961] [Q&A] [answered]
Hi,
Is it possible to report error in ports and, if so, where?
My problem is when installing a port, it starts to build , and after a while install fails due to a build error.

Is there a ports maintainer?
Should I look in the git log and contact the ports committer?

Or am I supposed to solve the problem myself, and submit a PR?

I can't find any information in the documents. Only how to create a ports.

Thanks for any hints!

PS. More specifically, I'm trying to `vcpkg install llvm[clang,libcxx] --triplet=x64-linux` on CentOS7 container using devtoolset-11, but  libcxx can't build because its stddef.h contains a `#include_next "stddef.h"` that can't be found. I guess this belongs to the more painful ports, however, in general is there any process for reporting ports problems? DS.

**Comment by Neumann-A** [comment, marked as answer]
#30604

Also sounds like an upstream problem? But you are free to submit a PR fixing it.
